src/Sarif.Driver/Sdk/MultithreadedAnalyzeCommandBase.cs

```diff
@@ -656,28 +656,42 @@ private async Task<bool> EnumerateFilesFromArtifactsProvider(TContext globalCont
 
                 string filePath = artifact.Uri.GetFilePath();
 
+                long artifactSizeInBytes = -1;
+                try
+                {
+                    artifactSizeInBytes = artifact.SizeInBytes.Value;
+                }
+                catch (Exception e) when (e is ArgumentException || (e is IOException && e.Message.IndexOf("The filename, directory name, or volume label syntax is incorrect.") != -1))
+                {
+                    // These exceptions can come out of the FileSystem when we're passing in invalid characters.  We need to catch and
```

Comment:
>exceptions [](http://example.com/codeflow?start=29&length=10)

NIT: please 'columnize this comment for better readability, i.e., prefer line lengths that are a bit closer to being equal in size.